dist/source/repo-creator/choose-repository.html

```diff
@@ -108,9 +106,16 @@ <h4 class="panel-title">
 							<div class="search-area" onkeypress="event.keyCode==13?document.getElementById('searchBtn').click():''">
 								<div class="input-group"">
 									<input type="text" class="form-control search-box" placeholder="GitHub Search Terms." value.two-way="searchInput"/>
-									<span class="input-group-addon search-button">
-										<a href="" id="searchBtn" type="submit" click.trigger="search()" disabled.bind="inputValidated"><img src="resources/icons/Search.svg" height="36" disabled.bind="inputValidated"/></a>
-									</span>
+										<span class="input-group-addon search-button">
+											<a href="" id="searchBtn" type="submit" click.trigger="search()" disabled.bind="inputValidated">
+											  <img src="resources/icons/Search.svg" height="36" disabled.bind="inputValidated"/>
```

Comment:
This line has tabs and spaces.  I believe only the last 2 characters are spaces, the rest are tabs.